src/types/matching.ts

```diff
@@ -0,0 +1,12 @@
+interface FeedbackType {
+  isNice: boolean;
+  isFluent: boolean;
+  isFunny: boolean;
+  isBad: boolean;
+}
+
+export interface MatchingFeedbackForm {
+  feedbackType: FeedbackType;
+  feedbackMessage: string;
+  neverMatchingAgain: boolean;
+}
```

Comment:
p3;
네이밍 수정해봤습니다. 확인부탁드려요!

```suggestion
export interface MatchingFeedbackForm {
  matchingFeedback: MatchingFeedback;
  message: string;
  isBlockedMatching: boolean;
}
```

Reply:
감사합니다~! 변수명 짓는거 너무 어렵네요 😂

Reply:
`MatchingFeedback`이라는 변수명은 피드백 유형이라는 의미 보다 조금 더 넓은 의미를 갖는다고 생각해, `FeedbackType`으로 하였습니다!

해당 변수명을 제외하고는 의견주신 네이밍으로 반영했습니다. 의견주셔서 감사합니다!

작업 커밋 : 1c0bd0ecfecbf617282d474041f2d986dae29ad2